spec_test.go

```diff
@@ -0,0 +1,37 @@
+package dalec
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/goccy/go-yaml"
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/assert/cmp"
+)
+
+func TestDate(t *testing.T) {
+	expect := "2023-10-01"
+	expectTime, err := time.Parse(time.DateOnly, expect)
+	assert.NilError(t, err)
+
+	d := Date{Time: expectTime}
+	assert.NilError(t, err)
```

Comment:
Is this assertion needed? I don't see we are recapturing `err` after last assertion.
```suggestion
```

Reply:
Fixed.